system-contracts/contracts/Constants.sol

```diff
@@ -1,6 +1,7 @@
 // SPDX-License-Identifier: MIT
+
 // We use a floating point pragma here so it can be used within other projects that interact with the zkSync ecosystem without using our exact pragma version.
-pragma solidity ^0.8.20;
+pragma solidity ^0.8.24;
```

Comment:
Btw, maybe we shouldn't limit constant pragma so strictly? There seems to be no reason to require at least `0.8.24`